File: Python/bubbleSort.py
#Bubble Sort Function
def bubble_sort(a):
    #looping through Array
    for i in range(len(arr)-1):
        for j in range(len(arr)-1-i):
            #swap if current element is greater than next element
            if a[j]>a[j+1]:
                a[j],a[j+1]=a[j+1],a[j]
    return a
#driver Code
#insert space separeted input
arr = list(map(int, input().split()))
#array Before Sorting
print("Before sorting",arr)
#array after Sorting
print("After Sorting",bubble_sort(arr))

File: JavaScript/bubbleSort.js
// bubble-Sort function
function bubbleSort(ar) {
    //initializing temp variable to swap
    let temp;
    //looping through array
    for (let i = 0; i < ar.length-1; i++) {
        for (let j = 0; j < ar.length-i-1; j++) {
            if (ar[j]>ar[j+1]) {
                //swapping array elements if current element is greater than next element
                // [ar[j],ar[j+1]]=[ar[j+1],ar[j]]  -->can use this swapping technique as well
                temp=ar[j] ;
                ar[j]=ar[j+1];
                ar[j+1]=temp;

            }            
        }
        
    }
    return ar;
}

//Driver Code

//taking input
arr=[5,4,3,2,1];
//printing array before Sorting
console.log("array before sorting:",arr)
swapped_array=bubbleSort(arr);
//printing array after sorting
console.log("array after sorting:",swapped_array);